.github/workflows/ci_ubuntu20_04_ros1_noetic.yml

```diff
@@ -25,4 +25,4 @@ jobs:
       - uses: ros-tooling/action-ros-ci@v0.2
         with:
           package-name: linear_feedback_controller_msgs
-          target-ros1-distro: noetic
+          target-ros2-distro: humble
```

Comment:
Maybe you should check if the action-ros-ci@v0.2 is appropriate for humble.

Reply:
yes there are much more recent versions I found out and furthermore I want to launch the build on humble/iron/jazzy. So I will fix this.